components/CourseList.tsx

```diff
@@ -1,39 +1,13 @@
-import { Box, Typography } from "@mui/material";
-import CircularProgressBar from "./CircularProgressBar";
-
 type Course = {
-  name: string;
-  title: string;
-  description: string;
-  // TODO
-  link?: string;
-  // number from 0 to 100
-  similarity: number;
-  easy: number;
-  useful: number;
+  courseCode: string;
+  courseName: string;
+  courseDescription: string;
+  similarityScore: number;
+  easyRating: number;
+  usefulRating: number;
```

Comment:
We dont actually have numbers for this, it would have to be pulled from UW flow. I think our decision was to remove these two